### PR TITLE
Write hello to file3

### PR DESCRIPTION
**Pull request by bhmdatcom**
Please check

### CODE REVIEW

File: file3.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (javierini) [inline file3.txt:1]: This line is good but it is better if you capiyulized H.

**Review by javierini — COMMENTED** [review]
Please make these changes

File: file3.txt (hunk 1 above)
Comment (javierini) [inline file3.txt:1]: use H

**Review by javierini — CHANGES_REQUESTED** [review]
make new push

**Review by bhmdatcom — COMMENTED** [review]
Eveything looks ok. lets go ahead  vid merging

**Review by javierini — APPROVED** [review]

**Pull request merged**